src/sentry/analytics/events/ai_autofix_pr_opened.py

```diff
@@ -0,0 +1,16 @@
+from sentry import analytics
+
+
+class AiAutofixPrOpenedEvent(analytics.Event):
+    type = "ai.autofix.pr.opened"
+
+    attributes = (
```

Comment:
Instead of having this 3 times you can follow this pattern: https://github.com/getsentry/sentry/blob/master/src/sentry/analytics/events/comment_webhooks.py